examples/energy-management-app/energy-management-common/src/DEMTestEventTriggers.cpp

```diff
@@ -0,0 +1,401 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
```

Comment:
supernit: new files should probably be 2024.

Reply:
Changed....